tests/tests_geomstats/test_special_linear.py

```diff
@@ -0,0 +1,151 @@
+"""Unit tests for the special linear group and traceless matrices."""
+
+
+import random
+
+import geomstats.backend as gs
+from geomstats.geometry.special_linear import SpecialLinear, TracelessMatrices
+from tests.conftest import TestCase
+from tests.data_generation import _LieGroupTestData, _MatrixLieAlgebraTestData
+
+# from tests.parametrizers import LieGroupParametrizer, MatrixLieAlgebraParametrizer
+
+
+# TODO: very similar to GeneralLinear tests (simplify common points)
+
+
+class TestSpecialLinear(TestCase):
+    space = group = SpecialLinear
+
+    # skips due to tolerance issues
+    skip_test_exp_log_composition = True
+    skip_test_log_exp_composition = True
+
+    class TestDataSpecialLinear(_LieGroupTestData):
```

Comment:
Hard/Expert (you can, but don't have to, address it): put this class into a special file within the tests/data folder and adapt the code accordingly. You can get inspiration on how it is done for other classes in geomstats: we have two files: one test file, and one data file.